src/components/FavoritesView/FavoritesView.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import {bindActionCreators} from "redux";
+import {Container} from "../../App.styled";
+import SearchBar from "../SearchBar/SearchBar";
+import GifList from "../GifList/GifList";
+import favoritesGif from '../../actions/favoritesGifs';
+
+export class FavoritesView extends Component {
+
+  componentDidMount() {
+    this.props.filterGifsClear();
+  }
+  render() {
+    const {allGifs, filtered} = this.props.favoritesGifs;
```

Comment:
you should take this filtering outside the render function to make it more readable